static/include/components/apis/generated/generic_component.md

```diff
@@ -111,61 +125,6 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 {{% /tab %}}
 {{< /tabs >}}
 
-### DoCommand
```

Comment:
Is this meant to be deleted?

Reply:
If you expand this file above, you can see it's here twice actually.